te_markdown/terminology.md

```diff
@@ -35,25 +35,47 @@ In this area, details about the test step selected in the _Test Execution Naviga
+: A file used by and / or created with the [Test-Editor-Web](#test-editor-web) for the overall purpose of user acceptance testing. In particular, these include files containing [test specifications](#test-specification), [test cases](#test-case), and [application mappings](#application-mapping) [test case language (TCL)](#test-case-language).
+
+<a name="test-case"></a>Test Case
+: A user acceptance test defined in terms of a sequence of [test steps](#test-step) to be executed, and expressed using the [test case language (TCL)](#test-case-language).
+
+<a name="test-case-language"></a>Test Case Language (TCL)
+: A language used by [testers](#tester) to implement [test cases](#test-case) conforming to [test specifications](#test-specification), by defining the necessary, concrete [test steps](#test-step) for automatic execution.
+
+<a name="test-editor-web"></a>Test-Editor-Web
+: An application dedicated to automate the tip of the testing pyramid—acceptance testing—to enable automated product releases and provide confidence in reliable product operation. Designed to bridge the gap between [domain experts](#domain-expert), [testers](#tester), and [developers](#developer), Test-Editor-Web is a modern application that integrates well with container technologies. See (Overview)[#overview] for a description of its user interface components and functions.
+
+<a name="test-fixture"></a>Test Fixture
+: A software library that implements concrete actions, specific to the platform and technology stack of a given [software under test](#software-under-test), to be performed during test execution. [Developers](#developer) use [application mappings](#application-mapping) to bind [test-steps](#test-step) to test fixture logic. Out of the box, [Test-Editor-Web](#test-editor-web) comes with fixtures that enable the testing of _web_ applications, in particular _Angular_ applications, _REST_ services, _Java Swing_ applications, and mainframe applications through _3270 terminal emulation_.
+
+<a name="test-specification"></a>Test Specification
+: A usage scenario describing an interaction with the [software under test](#software-under-test), written by [domain experts](#domain-expert) to express desired system behaviors in a testable manner.
 
 <a name="test-specification-language"></a>Test Specification Language (TSL)
-: TBW
--->
+: A language used by [domain experts](#domain-expert) to write [test specifications](#test-specification) of desired system behaviors. [Testers](#tester) refer to TSL [artifacts](test-artifact) when implementing them as [test cases](#test-case), using the [test case language](#test-case-language).
+
+<a name="test-step"></a>Test Step
+: A single, automatable step taken as part of a test executed against the [software under test](#software-under-test). In [test cases](#test-case) conforming to [test specifications](#test-specification), each [specification step](#specification-step) may be implemented by arbitrarily many test steps.
```

Comment:
```suggestion
: A single, automatable step taken as part of a test executed against the [software under test](#software-under-test). In [test cases](#test-case) conforming to [test specifications](#test-specification), each [specification step](#specification-step) may be implemented by arbitray many test steps.
```

Reply:
I assume you meant to write "arbitrary", but I think we should use an adverb here (even though "many" isn't really a verb – but its also not a noun, obviously, so I don't know…).